File: mint-location/src/main/java/net/crystalapps/mint/location/library/callbacks/InitializationCallback.java
package net.crystalapps.mint.location.library.callbacks;

/**
 * Created by Syed Owais Ali on 10/21/2018.
 */
public interface InitializationCallback {
    void onSuccess();
    default void onFailed(String error){}
    default void onPermissionDenied(){}
}

File: mint-location/src/main/java/net/crystalapps/mint/location/library/callbacks/InitializationCallback.kt
package net.crystalapps.mint.location.library.callbacks

/**
 * Created by Syed Owais Ali on 10/21/2018.
 */
interface InitializationCallback {
    fun onSuccess()
    fun onFailed(error: String) {}
    fun onPermissionDenied() {}
}
